extensions/kafka/src/main/java/io/deephaven/kafka/KafkaTools.java

```diff
@@ -138,6 +138,12 @@ public class KafkaTools {
     public static final String OFFSET_COLUMN_NAME_DEFAULT = "KafkaOffset";
     public static final String TIMESTAMP_COLUMN_NAME_PROPERTY = "deephaven.timestamp.column.name";
     public static final String TIMESTAMP_COLUMN_NAME_DEFAULT = "KafkaTimestamp";
+    public static final String RECEIVETIME_COLUMN_NAME_PROPERTY = "deephaven.receivetime.column.name";
+    public static final String RECEIVETIME_COLUMN_NAME_DEFAULT = null;
```

Comment:
Let's be consistent... if we're going to say "KEY_BYTES", we should say "RECEIVE_TIME".
```suggestion
    public static final String RECEIVE_TIME_COLUMN_NAME_PROPERTY = "deephaven.receivetime.column.name";
    public static final String RECEIVE_TIME_COLUMN_NAME_DEFAULT = null;
```